vendor/github.com/intel/govmm/qemu/qmp.go

```diff
@@ -242,8 +245,19 @@ type MigrationStatus struct {
 	XbzrleCache  MigrationXbzrleCache     `json:"xbzrle-cache,omitempty"`
 }
 
+// SchemaInfo represents all QMP wire ABI
+type SchemaInfo struct {
+	MetaType string `json:"meta-type"`
+	Name     string `json:"name"`
+}
+
 func (q *QMP) readLoop(fromVMCh chan<- []byte) {
 	scanner := bufio.NewScanner(q.conn)
+	if q.cfg.MaxCapacity != 0 {
+		buffer := make([]byte, q.cfg.MaxCapacity)
```

Comment:
Since `MaxCapacity` can be negative, this could panic, so please change the test to:

```go
if q.cfg.MaxCapacity > 0 {
```

Reply:
ok, i have updated in intel/govmm#104